ui/00_message/ui/QuestReward_res.xml

```diff
@@ -210,18 +210,21 @@ Total Sale: &lt;VAL GOLD&gt;</translation>
   <message>
     <key>REWARD_MSG_REWARD_PARTS</key>
     <original>部位破壊報酬</original>
+    <translation>Site destruction reward.</translation>
```

Comment:
Pretty sure "Site" should be "Part" here, as in the parts you can break on big bosses in some of the GMs.